runbotics/runbotics-desktop/src/action/microsoft/excel/automation/cloud-excel.error-message.ts

```diff
@@ -16,6 +16,15 @@ export class CloudExcelErrorMessage {
             ${this.getCheckInputTip()}
         `;
     }
+
+    static setCellsIncorrectInput(): string {
+        return `
+            Start cell addresses has to include a column letter and a row number (e.g. A1).
```

Comment:
too long.
`StartCell field expects cell address e.g. A3.
Values field expects to specify nested lists. Each list represents a row and the elements inside the values in the corresponding column, e.g. [["A3 value", "B3 value"],["A4 value", "B4 value"]])`